evennia/contrib/tutorials/red_button/README.md

```diff
@@ -7,7 +7,7 @@ object with its own functionality and state tracking.
 
 Create the button with
 
-    create/drop button:tutorials.red_button.RedButton
+    create/drop button:contrib.tutorials.red_button.RedButton
```

Comment:
I think the original does work, since it's added to the base paths. But this change is arguable even clearer :)